### DISCUSSION TITLE
Answering on a closed issue ore merged PR

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#2070] [General]
Hello, 

if i want to answer to a closed issue or an already merged PR, schould I answer in the closed Issue /PR or schould I raise a new issue/discussion linking to the closed issue/PR??  

I recently answered to this PR: #2052

**Comment by brockfanning** [comment]
@christoph-LSN Sorry for the delay - yes a new discussion linking to the old discussion/PR sounds good.
